app/objects/c_ability.py

```diff
@@ -55,12 +55,15 @@ def __init__(self, ability_id, tactic=None, technique_id=None, technique=None, n
         self.executor = executor
         self.platform = platform
         self.payloads = payloads if payloads else []
-        self.parsers = parsers
-        self.requirements = requirements
+        self.parsers = parsers if parsers else []
+        self.requirements = requirements if requirements else []
         self.privilege = privilege
         self.timeout = timeout
         self.repeatable = repeatable
-        self.variations = [Variation(description=v['description'], command=v['command']) for v in variations]
+        if variations:
+            self.variations = [Variation(description=v['description'], command=v['command']) for v in variations]
```

Comment:
You can follow the same format as above:
```Python
self.variations = [Variation(description=v['description'], command=v['command']) for v in variations] if variations else []
```

Reply:
Fixed in 083b1cb